### PR TITLE
Fix compile warnings

### PR DESCRIPTION
**Pull request by SchumBlubBlub**

### CODE REVIEW

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -101,7 +101,9 @@ static void cmd_pCD(RCore *core, const char *input) {
 		}
 		char *dis = r_core_cmd_str (core, cmd);
 		C(write)(c, dis);
-		free (cmd);
+		if (cmd) {
```
Comment (radare) [inline libr/core/cmd_print.c]: Unnecessary. Free(NULL) is completely valid

File: libr/core/cmd_write.c [hunk 2]
```diff
@@ -541,7 +541,9 @@ static int cmd_write(void *data, const char *input) {
 			r_core_write_at (core, core->offset, buf, len);
 			WSEEK (core, len);
 			r_core_block_read (core);
-			free (buf);
+			if (buf) {
```
Comment (radare) [inline libr/core/cmd_write.c]: Same here

File: libr/core/cmd_print.c (hunk 1 above)
Comment (SchumBlubBlub) [inline libr/core/cmd_print.c]: Removed.

File: libr/core/cmd_write.c (hunk 2 above)
Comment (SchumBlubBlub) [inline libr/core/cmd_write.c]: Removed.

**Pull request merged**